maze_transformer/generation/latticemaze.py

```diff
@@ -107,13 +93,16 @@ class LatticeMaze:
     Note: the bottom row connections going down, and the
     right-hand connections going right, will always be False.
     """
+
     connection_list: NDArray["lattice_dim x y", bool]
     generation_meta: dict | None = None
 
     grid_shape = property(lambda self: self.connection_list.shape[1:])
 
     n_connections = property(lambda self: self.connection_list.sum())
 
+    lattice_dim: int = 2
```

Comment:
Old location for lattice_dim had an error because it has a default value so it needs to be after all the parameters without default values

Reply:
`kw_only` is set to true, so I'm confused why this is happening. If this works though, then this is fine.

Reply:
Yeah I don't know enough about python config to know why but error went away when I moved it :D